src/lib/ng-package/schema.ts

```diff
@@ -0,0 +1,67 @@
+import * as ajv from 'ajv';
+import { NgPackageConfig } from '../../ng-package.schema';
+import * as log from '../utils/log';
+
+/** Lazily initialized ajv validator instance. */
+let ajvValidator: ajv.ValidateFunction | null = null;
```

Comment:
Personal preference

```suggestion
let ajvValidator: ajv.ValidateFunction | undefined;
```

Reply:
Oh, interesting; I personally much prefer the explicit initializer and explicit non-null check.

Reply:
You can leave it :)